Automattic/woocommerce-payments
# Ensure WooPay Preview Matches The Actual Implementation

Fixes 2559-gh-Automattic/woopay.

#### Changes proposed in this Pull Request









The changes in this PR ensure that the WooPay Checkout preview in the admin dashboard appears similar to the actual implementation.

#### Testing instructions





**Test: Confirm WooPay preview does not match the implementation**
1. Ensure you are on the `develop` branch.
2. As a shopper, navigate to the Merchant Shop, add an item to your cart, and proceed to checkout through WooPay.
3. Note what the WooPay Checkout design looks like.
4. As a merchant, navigate to `Payments > Settings`.
5. Scroll down to the `Express checkouts` section and click the `Customize` button for `WooPay`.
6. Scroll down to the `Preview of checkout` section and note that the preview appears different than the WooPay Checkout implementation.

**Test: Confirm WooPay preview matches the implementation**
1. Ensure you are on the `fix/2559-woopay-preview-implementation-mismatch` branch.
2. As a shopper, navigate to the Merchant Shop, add an item to your cart, and proceed to checkout through WooPay.
3. Note what the WooPay Checkout design looks like.
4. As a merchant, navigate to `Payments > Settings`.
5. Scroll down to the `Express checkouts` section and click the `Customize` button for `WooPay`.
6. Scroll down to the `Preview of checkout` section and note that the preview appears similar to the WooPay Checkout implementation.

**Test: Confirm the WooPay preview matches the designs**
1. Ensure you are on the `fix/2559-woopay-preview-implementation-mismatch` branch.
2. Open the designs made available in this comment: 2559-gh-Automattic/woopay#issuecomment-2046977690.
3. As a merchant, navigate to `Payments > Settings`.
4. Scroll down to the `Express checkouts` section and click the `Customize` button for `WooPay`.
5. Scroll down to the `Preview of checkout` section and note the preview.
6. Ensure the preview matches the designs.

**Test: Confirm the WooPay preview displays the site's logo, when available**
1. Ensure you are on the `fix/2559-woopay-preview-implementation-mismatch` branch.
2. As a merchant, navigate to `Appearance > Customize > Site Identity`, add a site logo, and click the `Publish` button.
3. Navigate to `Payments > Settings`.
4. Scroll down to the `Express checkouts` section and click the `Customize` button for `WooPay`.
5. Scroll down to the `Preview of checkout` section and note the preview.
6. Ensure a `Checkout logo` is not set and that the preview displays the site logo from Step 2.
7. Click the `Upload custom logo` button, upload a new logo, and ensure that the newly uploaded logo is displayed in the preview.
8. Remove the `Checkout logo` from Step 7 and the `Site logo` from Step 2.
9. Ensure the store's name is displayed in the preview.

**Test: Confirm the WooPay preview has a responsive design**
1. Ensure you are on the `fix/2559-woopay-preview-implementation-mismatch` branch.
2. As a merchant, navigate to `Payments > Settings`.
3.  Scroll down to the `Express checkouts` section and click the `Customize` button for `WooPay`.
4. Scroll down to the `Preview of checkout` section and note the preview.
5. Resize the window to emulate widescreen, desktop, tablet, and mobile widths.
10. Ensure the implementation maintains its design structure.

-------------------


**Post merge**
